### PR TITLE
Update Move-ADObject.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a statement about password when it moves within the object is moved across domains in the forest

Fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3514

Also added fixes for description which fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3540

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: docset/winserver2012-ps/activedirectory/Move-ADObject.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ You can also use the Get-ADGroup, Get-ADUser, Get-ADComputer, Get-ADServiceAccou
 The TargetPath parameter must be specified.
 This parameter identifies the new location for the object or container.
 
+The Cmdlet also moves the password when a user or computer object is moved across domains within a forest.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Move-ADObject.md:33]:
```suggestion
The cmdlet also moves the password when a user or computer object is moved across domains within a forest.
```

File: docset/winserver2012r2-ps/activedirectory/Move-ADObject.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ You can also use the Get-ADGroup, Get-ADUser, Get-ADComputer, Get-ADServiceAccou
 
 The **TargetPath** parameter must be specified.
 This parameter identifies the new location for the object or container.
+The Cmdlet also moves the password when a user or computer object is moved across domains within a forest.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/activedirectory/Move-ADObject.md:34]:
```suggestion
The cmdlet also moves the password when a user or computer object is moved across domains within a forest.
```

File: docset/winserver2016-ps/activedirectory/Move-ADObject.md [hunk 3]
```diff
@@ -37,6 +37,7 @@ You can also use the **Get-ADGroup**, **Get-ADUser**, **Get-ADComputer**, **Get-
 The *TargetPath* parameter must be specified.
 This parameter identifies the new location for the object or container.
 
+The Cmdlet also moves the password when a user or computer object is moved across domains within a forest.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/activedirectory/Move-ADObject.md:40]:
```suggestion
The cmdlet also moves the password when a user or computer object is moved across domains within a forest.
```

File: docset/winserver2019-ps/activedirectory/Move-ADObject.md [hunk 4]
```diff
@@ -36,6 +36,7 @@ You can also use the **Get-ADGroup**, **Get-ADUser**, **Get-ADComputer**, **Get-
 
 The *TargetPath* parameter must be specified.
 This parameter identifies the new location for the object or container.
+The Cmdlet also moves the password when a user or computer object is moved across domains within a forest.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/activedirectory/Move-ADObject.md:39]:
```suggestion
The cmdlet also moves the password when a user or computer object is moved across domains within a forest.
```

File: docset/winserver2022-ps/activedirectory/Move-ADObject.md [hunk 5]
```diff
@@ -37,6 +37,9 @@ You can also use the **Get-ADGroup**, **Get-ADUser**, **Get-ADComputer**, **Get-
 The *TargetPath* parameter must be specified.
 This parameter identifies the new location for the object or container.
 
+The Cmdlet also moves the password when a user or computer object is moved across domains within a forest.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/activedirectory/Move-ADObject.md:40]:
```suggestion
The cmdlet also moves the password when a user or computer object is moved across domains within a forest.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, please commit the two suggested changes that you seem to have missed. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**